pkg/mq/msgstream/mq_msgstream.go

```diff
@@ -567,74 +593,80 @@ func (ms *MqTtMsgStream) bufMsgPackToChannel() {
 		case <-ms.ctx.Done():
```

Comment:
It seem that the outside `select {case: default:}` do not make any sense here.
check of ms.ctx.Err() is already applied in function `continueBuffering`.

Reply:
this select meant to quick function earlier in case of closing msgstream to avoid the first run in buffer data loop